### PR TITLE
You can now examine megaphones to see if they're recharging. Useful when you wanna scream shit at people but you dont wanna say it softly.

### PR DESCRIPTION
**Pull request by alexkar598**
### Intent of your Pull Request

You can now examine megaphones to see if they're recharging. 

### Why is this change good for the game?

Useful when you wanna scream shit at people but you dont wanna say it softly.

# Wiki Documentation

### Briefly describe your PR and the impacts of it, in layman's terms. 
You can now examine megaphones to see if they're recharging. 

### What should players be aware of when it comes to the changes your PR is implementing?
You can now examine megaphones to see if they're recharging. 

### What general grouping does this PR fall under? 
Megaphones

# Changelog

:cl:  
rscadd: You can now examine megaphones to see if they're recharging. 
/:cl:

### CODE REVIEW

**Comment by redmoogle** [comment]
add a indicator light 4head

**Comment by alexkar598** [comment]
If youre not happy with this PR feel free to make an alternative.

**Comment by Bibby0110** [comment]
don't see why we can't have both honestly,

not sure everyone is going to be able to notice the singular pixel overlay

**Comment by redmoogle** [comment]
> 
> 
> don't see why we can't have both honestly,
> 
> not sure everyone is going to be able to notice the singular pixel overlay

I mean yeah, but whos gonna examine the thing every single time

**Comment by alexkar598** [comment]
Me
